{{ cookiecutter.repo_name }}/README.md

```diff
@@ -23,6 +23,14 @@ using:
 pip install -e .
```

Comment:
Do we need the pip install here or should it be make install?

Also there is a bit at the bottom of the README.md which say about doing pip install requirements. Can this be replaced / removed and can you check in the other docs it's not there

Reply:
Replaced the requirements at the bottom to use the makefile. I've left pip install as an option as well as the Makefile. Do you think I should just keep the Makefile install?